docs/UserGuide.md

```diff
@@ -65,107 +81,122 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
-
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
-
-### Locating persons by name: `find`
-
-Finds persons whose names contain any of the given keywords.
-
-Format: `find KEYWORD [MORE_KEYWORDS]`
-
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
+_Adds a person to the address book._
+>> add n/NAME e/EMAIL y/YEAR c/COURSE [OPTIONAL/PARAMETER]...
+> 
+> **Compulsory parameters:**
+> - n/NAME (eg. n/John Doe)
+> - e/EMAIL (eg. e/johndoe123@email.com)
+> - y/YEAR (eg. ay/3, ay/prof, ay/alum)
+> - c/COURSE (eg. c/Computer Science, c/Information Security)
+> 
+> **Optional parameters:**
+> - ct/CONTACT (eg.ct/Telegram: @Johndoe123)
+> - sn/STUDENT NUMBER (eg. sn/E1234568)
+> - m/MODULE YEAR (eg. m/AY2223S1 CS2103T, m/AY2122S2 CS2101)
+> - job/WORK (eg. job/Google Software Engineer, job/ATAP)
+> - skill/SKILL (eg. skill/C++, skill/Rust)
+> - link//LINK (eg. link//github.com/johndoe123)
+> 
+> 💡 **Tip:** A person can have any number of [OPTIONAL/PARAMETER] (including 0)
+> 
+> **Examples:**
+> - `add` n/Betsy Crowe e/betsycrowe@example.com y/prof c/Information Security ct/Telegram: @betC123 job/Professor, NUS IS link//linkedin.com/in/betsycrowe 
+> - `add` n/John Doe e/johnd@example.com y/2 c/Computer Science
+> ![Add Example](images/UiAddContact.png)
+
+
+<a name="view"></a>
+### Viewing contacts/tabs: `view`
+_View more details of specific contact._
+>> view INDEX
+> 
+> Loads up the contact at INDEX to the right panel to view details
+> 
+> ![image](images/UIViewIndex.png)
+
+>> view TAB
+> 
+> Switches the tab of the right panel (eg. view c, view w, view m)
+> 
+> ![image](images/UiViewTab.png)
+
+
+<a name="edit"></a>
+### Editing a person information: `edit`
+_Edits an existing person in the address book._
+>> edit [n/NAME] [e/EMAIL] [y/YEAR] [c/COURSE] [OPTIONAL/PARAMETER]...
+> 
+> - Edits the person displayed in the view panel 
+> - At least one of the optional fields must be provided. 
+> - Existing values will be updated to the input values. 
+> - When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative. 
+> - You can remove all the person’s tags by typing t/ without specifying any tags after it
+> 
+> **Examples:**
+> - edit p/91234567 e/johndoe@example.com will Edits the phone number and email address of the person to be 91234567 and johndoe@example.com respectively. 
```

Comment:
there is a grammar mistake in this sentence. Maybe you can change it to `edits` instead of `will Edits`!
Maybe you can also wrap this command as well as other commands in the file into a code format using <ctrl + e>